### PR TITLE
JeOS to Minimal VM rename

### PR DESCRIPTION
**Pull request by dmpop**
### PR creator: Description

Rename the JeOS guide to the Minimal VM Guide

### PR creator: Are there any relevant issues/feature requests?

* jsc#SLE-21530

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: DC-SLES-minvm [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ## ----------------------------
```
Comment (tomschr) [inline DC-SLES-minvm:1]: If I recall corrctly, due to SEO reasons, the DC files and file names should be kind of "speaking" for itself. Maybe use `DC-SLES-minimal-vm`? That would need to be applied also to IDs. Certainly Stefan knows more about this.

File: xml/MAIN.SLEDS.xml [hunk 2]
```diff
@@ -121,7 +121,7 @@
   <xi:include href="art_installation-sleds.xml"/>
   <xi:include href="art_modules.xml"/>
   <xi:include os="sles" href="art_virtualization-best-practices.xml"/>
-  <xi:include os="sles" href="art_jeos.xml"/>
+  <xi:include os="sles" href="art_minvm.xml"/>
```
Comment (tomschr) [inline xml/MAIN.SLEDS.xml:124]: Due to the above reasons, you should use `article_minimal_vm.xml`:

```suggestion
  <xi:include os="sles" href="article_minimal_vm.xml"/>
```

File: xml/art_minvm.xml [hunk 3]
```diff
@@ -5,10 +5,10 @@
 %entities;
 ]>
 <article xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
- xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="article-jeos"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="article-minvm"
```
Comment (tomschr) [inline xml/art_minvm.xml:8]: Same here:

```suggestion
 xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="article-minimal-vm"
```

File: xml/art_minvm.xml [hunk 4]
```diff
@@ -23,7 +23,7 @@
   </dm:docmanager>
   <abstract>
    <para>
-    The following document provides an overview of &sls; &jeos; and describes
+    The following document provides an overview of &sls; &minvm; and describes
```
Comment (tomschr) [inline xml/art_minvm.xml:26]: Would it help to add a references to the old name? Some customers may know the old name. Maybe something like "(formerly known as JeOS)"?

File: xml/art_minvm.xml [hunk 5]
```diff
@@ -97,24 +97,24 @@
...
-   <variablelist xml:id="vl-jeos-sles-diff">
-    <title>Differences between &jeos; and &sls;</title>
+   <variablelist xml:id="vl-minvm-sles-diff">
+    <title>Differences between &minvm; and &sls;</title>
     <varlistentry>
      <term>jeos-firstboot</term>
      <listitem>
       <para>
-       &jeos; comes with the <package>jeos-firstboot</package> tool, which can
+       &minvm; comes with the <package>jeos-firstboot</package> tool, which can
```
Comment (tomschr) [inline xml/art_minvm.xml:117]: It the name `jeos-firstboot` still the same? Or has the name changed as well?

File: DC-SLES-minvm [hunk 6]
```diff
@@ -1,11 +1,11 @@
 ## ---------------------------- 
 ## Doc Config File for SLES/SLED
-## JeOS Quick Start
+## Minimal VM Quick Start
 ## ----------------------------
 ##
 ## Basics
 MAIN="MAIN.SLEDS.xml"
-ROOTID=article-jeos
+ROOTID=article-minvm
```
Comment (tomschr) [inline DC-SLES-minvm:8]:
```suggestion
ROOTID=article-minimal-vm
```

**Review by sknorr — COMMENTED** [review]
Generally looks good!

However, unless "minvm" is going to be the official acronym, I'd avoid using it in ID(s) -- as it's against the style rule "Favor longer terms over non-obvious abbreviations."
I'd suggest `article-minimum-vm` (boring, predictable, both of which are good attributes).

I'd also like to ask you to hold the changes until we've merged the entity changes in #1069 (which is hopefully going to be tomorrow).

**Review by tomschr — COMMENTED** [review]
@dmpop I added some notes, just minimal things. Maybe Stefan can confirm or refute my comments.

File: xml/MAIN.SLEDS.xml (hunk 2 above)
Comment (sknorr) [reply xml/MAIN.SLEDS.xml:124]: I don't mind the xml file names as much -- unlike IDs, customers never see them. (Though making sure they're consistent might avoid future confusion of your co-workers.)

File: xml/art_minvm.xml (hunk 5 above)
Comment (sknorr) [reply xml/art_minvm.xml:117]: [This page](https://confluence.suse.com/display/jeos/jeos+packages+renamed) says it's still `jeos-firstboot`. Not sure if it's up-to-date.

Comment (tomschr) [reply xml/art_minvm.xml]: Thanks for the link. The last change was from 2022-02-02, so it's up-to-date.

Dima, the page was changed by Vincent, maybe ask him for clarification?

Comment (vmoutoussamy) [reply xml/art_minvm.xml]: it's still jeos-firstboot for 15SP4, we recently got feedback that MicroOS might use this in the future... so we might renamed it in the future (sic) but not for 15SP4

File: xml/art_minvm.xml (hunk 4 above)
Comment (dmpop) [reply xml/art_minvm.xml]: Good point. I've added the reference.

**Pull request merged**